packages/lib/src/vis/line/LineVis.module.css

```diff
@@ -6,15 +6,33 @@
   margin: 1rem;
 }
 
-.tooltipValue {
-  margin-top: 0.25rem;
+.tooltipItem {
+  display: flex;
+  align-items: baseline;
 }
 
-.tooltipValue:not(:last-child) {
-  margin-bottom: 0.25rem;
+.tooltipValue,
+.tooltipAux {
+  composes: tooltipItem;
 }
 
-.tooltipValue > strong {
+.tooltipValue,
+.tooltipValue + .tooltipAux {
+  margin-top: 0.25rem;
+}
```

Comment:
I get the impression it displays fine without this `margin-top` but up to you.

Reply:
Hmm, I find it aerates things a bit more.